### DISCUSSION TITLE
The problem with the "slash" symbol in the image path

### DISCUSSION THREAD

**Discussion by bcwang535** [zoni/obsidian-export#63] [Support and Q&A]
First of all thanks for this useful program, it saves me a lot of work, but now I have a problem about the slash in the path.

My original image link is "Wikilink" 

`![[Pasted image xxx.png]]`

and path of the image link exported by this program is backslash, the resulting path is as follows.

`![Pasted image xxx.png](_resources\Install-obsidian-export-record\Pasted%20image%20xxx.png)`

Then when I upload this md file to my server, the picture will not be displayed due to the wrong slash symbol,

the correct symbol should be written like this

`![Pasted image xxx.png](_resources/Install-obsidian-export-record/Pasted%20image%20xxx.png)`

I want to know if there is any way to adjust for this problem instead of manually adjusting one by one? 

Thank you~

**Comment by zoni** [comment]
Are you exporting your notes on Windows (where paths are separated by backslashes), then copying the resulting files to a Linux-based server (where files are separated by forward slashes)?

**Reply by trojblue** [reply]
I'm having the same issue here. Writing obsidian in windows, then publish to GitHub pages. Adding an argument to specify slash format would be really helpful.
